client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -119,14 +119,17 @@ class BusinessDetails extends Component {
 	}
 
 	getNumberRangeString( min, max = false, format = numberFormat ) {
+		// The localized strings below do not use sprintf to allow
```

Comment:
Unfortunately, I don't think this is going to end up working. I ran `npm run i18n` and looked at the resulting strings, and these just drop off the value:

```
#: languages/woocommerce-admin.php:2114
msgctxt "More than a certain revenue amount"
msgid "More than "
msgstr ""
```

I was originally thinking we would hard coded the string as

`More than $250,000` and the string could translate from there.. but you bring up a good point that the store language and currency may differ. Though since we don't currently ask for currency in the wizard I think this would be WC default anyway.

The revenue answer also isn't used inside the store, so I think it's OK that it's a difference currency, as long as it's in a format the user can understand without converting.

Reply:
Bummer- reverted back to the sprintf so this PR is now just adding a comment and the positional formatting that @jeffstieler mentioned below.